src/components/Organization/index.tsx

```diff
@@ -149,6 +153,11 @@ export default function OrganizationView(): JSX.Element {
             display={true}
           />
         </Grid>
+        {timeZoneFeatureEnabled && currentTimeZone && (
```

Comment:
The org will default to UTC in the BE.
We could still show the default here with a tooltip that it's a default?
The locations (seedbank, nursery, etc.) show the time zone value - explicit or inferred.
Not sure what's best.. we could either remove it from the other views or add it here.

Reply:
I can add a tooltip 👍 I like that pattern.